migrations/move_site_icons.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ *
+ * phpBB Browser Push Notifications. An extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2024, phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\webpushnotifications\migrations;
+
+use phpbb\db\migration\container_aware_migration;
+use phpbb\filesystem\exception\filesystem_exception;
+use phpbb\filesystem\filesystem;
+
+class move_site_icons extends container_aware_migration
+{
+	private const NEW_ICON_DIR = 'images/site_icons';
+	private const OLD_ICON_DIR = 'images/icons';
```

Comment:
Probably just add `OLD_ICON_DIR` to the `ext` class and then use both it and `PWA_ICON_DIR` here statically.

Reply:
In the migration I wanted to keep as much as possible self contained. OLD_ICON_DIR is only ever used here so there's no real reason to put it into the ext file. I thought about using PWA_ICON_DIR here for NEW_ICON_DIR, but then the migration would rely on our ext file which I want to avoid. Also, setting a constant with another constant seems wonky too `const NEW_ICON_DIR = PWA_ICON_DIR`